src/ansys/aedt/core/visualization/post/post_common_3d.py

```diff
@@ -498,7 +498,10 @@ def get_scalar_field_value(
             self.ofieldsreporter.CalcOp(scalar_function)
 
         if not variations:
```

Comment:
I see that you have this code snippet repeated in post_common_3d.py (3 times). Why don't you  create a private method? something like: 
```
def _set_variations_to_nominal(self):
    independent_flag = self._app.available_variations.independent
    self._app.available_variations.independent = True
    variations = self._app.available_variations.nominal_values
    self._app.available_variations.independent = independent_flag
    return variations
```
@Samuelopez-ansys